EnergyStar/EnergyStar.csproj

```diff
@@ -3,13 +3,30 @@
   <PropertyGroup>
     <OutputType Condition="'$(Configuration)'=='ReleaseInvisible'">WinExe</OutputType>
     <OutputType Condition="'$(Configuration)'!='ReleaseInvisible'">Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
     <Configurations>Debug;Release;ReleaseInvisible</Configurations>
   </PropertyGroup>
 
+  <!-- Optimize for size -->
+  <PropertyGroup Condition="'$(Configuration)'=='Release' or '$(Configuration)'=='ReleaseInvisible'">
+    <PublishAot>true</PublishAot>
+    <IlcOptimizationPreference>Size</IlcOptimizationPreference>
+    <IlcFoldIdenticalMethodBodies>true</IlcFoldIdenticalMethodBodies>
+    <IlcGenerateStackTraceData>false</IlcGenerateStackTraceData>
+    <TrimUnusedDependencies>true</TrimUnusedDependencies>
+    <RootPackageReference>false</RootPackageReference>
+    <TrimMode>link</TrimMode>
```

Comment:
`TrimMode=link` has been deprecated in .NET 7. Use `TrimMode=full`instead.

Reply:
Updated